packages/manager/modules/dbaas-logs/src/logs/detail/home/home.module.js

```diff
@@ -0,0 +1,37 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import '@ovh-ux/ng-ovh-sidebar-menu';
```

Comment:
No peerdep required for `@ovh-ux/ng-ovh-sidebar-menu` ? :)

Reply:
modules shouldn't need the sidebar menu, I'll remove the import